src/probabilities_estimators/histograms/rectangular_binning.jl

```diff
@@ -206,19 +179,89 @@ function RectangularBinEncoding(::AbstractDataset{D, T},
 
     RectangularBinEncoding(b, mini, edgelengths)
 end
+# This version exists if the given `ϵ`s are already tuples.
+# Then, the dataset doesn't need to be provided.
+function RectangularBinEncoding(b::FixedRectangularBinning{<:NTuple}, n_eps = 1)
```

Comment:
We should probably use `n_eps = 2` here too, unless we can guarantee that roundoff errors won't give an extra bin (which shouldn't be there).